runtime/modules/qrcode/qrcode.go

```diff
@@ -0,0 +1,126 @@
+package qrcode
+
+import (
+	"fmt"
+	"image/color"
+	"math/rand"
+	"strings"
+	"sync"
+	"time"
+
+	goqrcode "github.com/skip2/go-qrcode"
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+	"tidbyt.dev/pixlet/render"
+)
+
+const (
+	ModuleName = "qrcode"
+)
+
+var (
+	once   sync.Once
+	module starlark.StringDict
+)
+
+func LoadModule() (starlark.StringDict, error) {
+	once.Do(func() {
+		rand.Seed(time.Now().UnixNano())
+		module = starlark.StringDict{
+			ModuleName: &starlarkstruct.Module{
+				Name: ModuleName,
+				Members: starlark.StringDict{
+					"generate": starlark.NewBuiltin("generate", generateQRCode),
+				},
+			},
+		}
+	})
+
+	return module, nil
+}
+
+func generateQRCode(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var (
+		starUrl        starlark.String
+		starSize       starlark.String
+		starColor      starlark.String
+		starBackground starlark.String
+	)
+
+	if err := starlark.UnpackArgs(
+		"generate",
+		args, kwargs,
+		"url", &starUrl,
+		"size", &starSize,
+		"color?", &starColor,
+		"background?", &starBackground,
+	); err != nil {
+		return nil, fmt.Errorf("unpacking arguments for generate: %w", err)
+	}
+
+	// Validate size.
+	if !contains([]string{"small", "medium", "large"}, starSize.GoString()) {
+		return nil, fmt.Errorf("size must be small, medium, or large")
```

Comment:
Could be the `default:` of the switch below.

Reply:
I'd hate for a bad value to default though, which could end up being confusing to the user

Reply:
Ah! You mean error on default. I like it. One sec

Reply:
I meant that the `contains()` method is effectively already implemented by the switch statement. This check could be a default clause in the switch.